src/serviceFabricDiscoveryService/pkg/discovery/discoveryworker.go

```diff
@@ -142,7 +142,7 @@ func (p *Provider) loadConfiguration(ctx context.Context, cfgChan chan<- []byte)
 		case <-ticker.C:
 			e, err := p.fetchState()
 			if err != nil {
-				log.Print(err)
+				log.Printf("failed to gets applications %v", err)
```

Comment:
should the "failed to gets applications" log print remain in the same place and only be added if p.sfClient.GetApplications() returns an error